packages/extension-ui/src/Popup/Derive/Derive.spec.tsx

```diff
@@ -156,7 +156,9 @@ describe('Derive', () => {
       const button = wrapper.find('button[type="submit"]');
 
       expect(button.prop('disabled')).toBe(false);
-      expect(wrapper.find('.warning-message')).toHaveLength(0);
+      expect(
+        wrapper.findWhere((node) => node.prop('messageType') === 'critical' && node.prop('in') === true)
```

Comment:
I'm assuming this can't be more user-alike (aka less implementation testing) because it realtes to animations?

Reply:
I guess I could just iterate over a list of all messages - I made if as a - perhaps too close - copy of previous solution :thinking:

Reply:
Ok, a copy of previous solution is good enough actually :)